bin/build-dev

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+
+set -e
+set +x
+export AWS_DEFAULT_REGION=us-west-2
+REGION=us-west-2
+
+aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/t1q6b4h2
+docker build -t uat-dev --cache-from public.ecr.aws/t1q6b4h2/universal-application-tool:latest .
```

Comment:
Question (blocking): Does this need to be `public.ecr.aws/t1q6b4h2/uat-dev`?

Reply:
Either would work but your suggestion will work better if / as the two diverge!  Changed.